dev/manifests/kotsadm/deployment.yaml

```diff
@@ -6,28 +6,43 @@ metadata:
     kots.io/kotsadm: 'true'
     kots.io/backup: velero
 spec:
+  selector:
+    matchLabels:
+      app: kotsadm
   template:
     metadata:
       labels:
+        app: kotsadm
         kots.io/kotsadm: 'true'
         kots.io/backup: velero
       annotations:
         backup.velero.io/backup-volumes: backup
         pre.hook.backup.velero.io/command: '["/bin/bash", "-c", "curl -f http://kotsadm:password@kotsadm-rqlite:4001/db/backup -o /backup/kotsadm-rqlite.sql"]'
         pre.hook.backup.velero.io/timeout: 3m
     spec:
-      serviceAccountName: default
+      restartPolicy: Always
+      serviceAccountName: kotsadm
       containers:
         - name: kotsadm
+          image: kotsadm-api-dev
+          imagePullPolicy: Never
+          ports:
+            - name: http
+              containerPort: 3000
+            - name: dlv
+              containerPort: 30001
+          resources:
+            limits:
```

Comment:
any reason to limit this in dev?

Reply:
yes, to mimic prod as there were times where that bit us. i guess we remove the limits on "up" so that might be sufficient?